### PR TITLE
Fix: CREATE MODEL option can not take integer value

### PR DESCRIPTION
**Pull request by da-tubi**
Fix https://github.com/eto-ai/rikai/issues/222

### CODE REVIEW

**Comment by da-tubi** [comment]
The failed CI: timeout on an unrelated python tests.

File: src/test/scala/ai/eto/rikai/sql/spark/execution/CreateModelCommandTest.scala [hunk 1]
```diff
@@ -39,7 +39,7 @@ class CreateModelCommandTest extends AnyFunSuite with SparkTestSession {
     spark
       .sql(
         "CREATE MODEL qualified_opt OPTIONS (" +
-          "rikai.foo.bar=1.23, foo='bar', num=-1.23, flag=True) " +
+          "rikai.foo.bar=1.23, foo='bar', num=-1.23, batch=10, flag=True) " +
```
Comment (eddyxu) [inline src/test/scala/ai/eto/rikai/sql/spark/execution/CreateModelCommandTest.scala:42]: thanks for adding the test!

File: src/main/antlr4/org/apache/spark/sql/ml/parser/RikaiExtSqlBase.g4 [hunk 2]
```diff
@@ -153,6 +153,10 @@ fragment DIGIT
     : [0-9]
     ;
 
+fragment NONDIGIT
```
Comment (eddyxu) [inline src/main/antlr4/org/apache/spark/sql/ml/parser/RikaiExtSqlBase.g4:156]: the sql parser is case insensitive.  

https://github.com/eto-ai/rikai/blob/master/src/main/scala/ai/eto/rikai/sql/spark/parser/RikaiExtSparkSQLParser.scala#L86

Maybe just make `IDENTIFER` to be:

```
IDENTFIER：(LETTER | '_') (LETTER|DIGIT| '_')* ;
```

**Review by eddyxu — APPROVED** [review]
LGTM. Thanks

**Pull request merged**